addons/common/functions/fnc_errorMessage.sqf

```diff
@@ -7,7 +7,7 @@
  *
  * Arguments:
  * 0: Header <STRING>
- * 1: Text <STRING|TEXT>
+ * 1: Text <STRING or TEXT>
```

Comment:
isn't `STRING or TEXT` redundant?

Reply:
TEXT refers to [Structured Text](https://community.bistudio.com/wiki/Structured_Text)